src/main/io/gps.c

```diff
@@ -878,30 +1364,40 @@ void gpsUpdate(timeUs_t currentTimeUs)
 
         case GPS_STATE_LOST_COMMUNICATION:
             gpsData.timeouts++;
-            if (gpsConfig()->autoBaud) {
+            // commented out, as it is dangerous and should not be used, fix for a correct auto baud in another pr
+            /*if (gpsConfig()->autoBaud) {
```

Comment:
Please remove commented out code throughout your PR - the codebase is already hard to follow, but these commented out pieces are impossible to interpret

Reply:
generally, this will happen.

Reply:
I'm not following - you can add a comment *in a different PR* not to use autoBaud (which is a feature now and people use it in Configurator as well). For the current PR it's already too big for one, let's keep it focused and either not touch autoBaud, or if you must touch it, leave a comment (without code) that autoBaud is fragile and has been removed.

Reply:
Comment updated, old code removed.
Takes me time to get around to it.